system-tests/package.json

```diff
@@ -18,7 +18,7 @@
     "test/**/*"
   ],
   "dependencies": {
-    "@keep-network/tbtc-v2.ts": "2.1.0-dev.0",
+    "@keep-network/tbtc-v2.ts": "2.2.0-dev.0",
```

Comment:
Please use `^2.2.0`

Reply:
Done in a9be14bc29e359b0d7e6e652932b68da2d06a8b1.